capitalize_decorator.rb

```diff
@@ -0,0 +1,7 @@
+require './decorator'
+
+class CapitalizeDecorator < Decorator
+  def correct_name
+    @nameable.correct_name
```

Comment:
- [ ] I think you should capitalize the first letter of the given name, the code can be similar to the example below 👇 👇 

eg:
```ruby
class CapitalizeDecorator < Decorator
   def correct_name
     @nameable.correct_name.capitalize
   end
end
```